scheduler/src/cook/passport.clj

```diff
@@ -0,0 +1,15 @@
+(ns cook.passport
+  (:require [clojure.data.json :as json]
+            [clojure.tools.logging :as log]
+            [congestion.limits :refer [RateLimit]]
+            [cook.config :as config]
+            [cook.rest.impersonation :refer [impersonation-authorized-wrapper]]
+            [plumbing.core :refer [fnk]]))
```

Comment:
Some of these are unused, aren't they?

Reply:
Ah yeah, good catch I meant to remove those before pushing but forgot to :/ my bad